project2/load-data/.gitignore

```diff
@@ -0,0 +1 @@
+articles/*
```

Comment:
Would it be nice to actually have the data in the repo to be completely transparent in the repo and show what data has been used to actually run our experiments?

Or would it be too large?

Reply:
It's 28GB decompressed + 7GB compressed, so way too much to store in GitHub 😅